docs/docs/en/guide/task/conditions.md

```diff
@@ -4,25 +4,25 @@ Condition is a conditional node, that determines which downstream task should ru
-  - Success: When the upstream task runs successfully, run the success branch.
-  - Failure: When the upstream task runs failed, run the failure branch.
-- Upstream condition selection: can select one or more upstream tasks for conditions.
-  - Add an upstream dependency: the first parameter is to choose a specified task name, and the second parameter is to choose the upstream task status to trigger conditions.
-  - Select upstream task relationship: use `and` and `or` operators to handle the complex relationship of upstream when there are multiple upstream tasks for conditions.
+## Task Parameters
+
+| **Parameter** | **Description** |
+| -------- | ---------- |
+| Node Name | Set the name of the task. Node names within a workflow definition are unique. |
+| Run flag | Indicates whether the node can be scheduled normally. If it is not necessary to execute, you can turn on the prohibiting execution switch. |
+| Description | Describes the function of this node. | 
+| Task priority | When the number of worker threads is insufficient, they are executed in order from high to low according to the priority, and they are executed according to the first-in, first-out principle when the priority is the same. |
+| Worker group | The task is assigned to the machines in the worker group for execution. If Default is selected, a worker machine will be randomly selected for execution. | 
+| Task group name | The group in Resources, if not configured, it will not be used. |
+| Environment Name | Configure the environment in which to run the script. |
+| Number of failed retries | The number of times the task is resubmitted after failure. It supports drop-down and manual filling. |
+| Failure Retry Interval | The time interval for resubmitting the task if the task fails. It supports drop-down and manual filling. |
+| Timeout alarm | Check Timeout Alarm and Timeout Failure. When the task exceeds the "timeout duration", an alarm email will be sent and the task execution will fail. |
+| Downstream tasks selection | Depending on the status of the predecessor task, you can jump to the corresponding branch, currently two branches are supported: success, failure <ul><li style="list-style-type: none;"><ul><li>Success: When the upstream task runs successfully, run the success branch.</li><li>Failure: When the upstream task runs failed, run the failure branch.</li></ul></li></ul> |
+| Upstream condition selection | Can select one or more upstream tasks for conditions.<ul><li style="list-style-type: none;"><ul><li>Add an upstream dependency: the first parameter is to choose a specified task name, and the second parameter is to choose the upstream task status to trigger conditions.</li><li>Select upstream task relationship: use<span>&nbsp;</span><code>and</code><span>&nbsp;</span>and<span>&nbsp;</span><code>or</code><span>&nbsp;</span>operators to handle the complex relationship of upstream when there are multiple upstream tasks for conditions.</li></ul></li></ul> |
```

Comment:
I find some of html syntax (`li`, `ul`, `span`) in this item, could you please remove them to markdown syntax?